lib/utils/authentication.js

```diff
@@ -9,20 +9,20 @@ const Mcrypt = require('mcrypt').MCrypt;
 const cypher = new Mcrypt('rijndael-128', 'ecb');
```

Comment:
That would be a good idea. I'll add a utility library `mcrypt-cypher.js` with methods `encrypt` and `decrypt`